apps/dashboard/src/components/data-table/table-cell-badge.tsx

```diff
@@ -38,7 +40,12 @@ export function TableCellBadge({
         {isTruncated ? (
           <Tooltip open={open} onOpenChange={setOpen}>
             <TooltipTrigger
-              onClick={(event) => event.preventDefault()}
+              onClick={(event) => {
```

Comment:
It's not a good a11y to add `onClick` events to navigate pages. Instead we should use `Link` or `a` components with proper href.

Let's keep this TableCellBadge as is and wrap the Link component everywhere.